sources/packages/api/src/services/education-program-offering/education-program-offering-import-csv.service.ts

```diff
@@ -166,15 +169,16 @@ export class EducationProgramOfferingImportCSVService {
       offeringModels.push(offeringModel);
       offeringModel.institutionLocationCode = line[CSVHeaders.locationCode];
       offeringModel.sabcProgramCode = line[CSVHeaders.sabcProgramCode];
-      offeringModel.offeringName = line[CSVHeaders.offeringName];
+      offeringModel.offeringName = line[CSVHeaders.offeringName]?.trim();
```

Comment:
Why trim only for offering name and WILComponentType?

Reply:
All other ones have some sort of validation that will enforce the proper values. These ones can receive a string with empty spaces and the validator @isnotempty will let it pass.